src/toil_lib/tools/variant_annotation.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from toil_lib.programs import docker_call
+
+
+def gatk_genotype_gvcfs(job, gvcf_ids, config, emit_threshold=10.0, call_threshold=30.0):
+    """
+    Runs GenotypeGVCFs on one or more gVCFs generated by HaplotypeCaller.
+
+    :param JobFunctionWrappingJob job: Toil Job instance
+    :param dict gvcf_ids: Dictionary of GVCF FileStoreIDs
+    :param Namespace config: Configuration options and shared FileStoreIDs
```

Comment:
Take a look at `run_bwakit`'s docstring — the `config` object is a black box of attributes that needs a comprehensive description so someone can use this function just by reading the docstring.